static/css/app.css

```diff
@@ -1321,11 +1321,37 @@ textarea:focus {
 #participantsTable th {
   border-top: none;
 }
+
 .no-table-data {
   font-style: italic;
   color: #8b8d9b;
 }
 
-#participantsTable th {
+#collectedDataTable th:nth-child(1) {
```

Comment:
you can also just try this. it was working for me..
`#collectedDataTable th {
  border-top: none;
}`